config/resources/slave-besluit-domain.lisp

```diff
@@ -228,10 +228,13 @@
   :class (s-prefix "ext:Intermission")
   :properties `((:started-at :datetime ,(s-prefix "prov:startedAtTime"))
                 (:ended-at :datetime ,(s-prefix "prov:endedAtTime"))
-                (:comment :string ,(s-prefix "rdfs:comment")))
+                (:comment :string ,(s-prefix "rdfs:comment"))
+                (:position :string ,(s-prefix "schema:position")))
```

Comment:
I think using the schema:position is not the best here as we use it as a number in other models. Maybe is a bit confusing that sometimes it's a number and sometimes a string. Have to check with someone else

Reply:
yeah I wasnt sure about the rdfa, so i just put the first thing that came to mind

Reply:
@nvdk

Reply:
I have been thinking about this? Maybe an extra entity would make sense in that case, because it's something like a complex position that needs a prefix and an agendapoint. If this was a relational db I would make another table that has the prefix and the key of the datapoint, maybe we should make it an entity that has a property prefix and a hasOne relationship to the agendapoint

Reply:
indeed schema:position is not a good choice here, given its definition:
> The position of an item in a series or sequence of items.

I quite like oscars suggested approach, it may be a bit more work to process later on, but semantically it makes a lot of sense in that these two datapoints together have a meaning.

Reply:
so like a concept/concept-scheme sorta deal? @nvdk

Reply:
@Asergey91 I was thinking of something like:
```
...
 :has-one `((intermission-time-period :via ,(s-prefix "ext:hasTimePeriod")
                      :as "position")
...
(define-resource intermission-time-period ()
  :class (s-prefix "ext:IntermissionTimePeriod")
  :properties `((:position ,(s-prefix "ext:position"))
  :has-one `((agendapoint :via ,(s-prefix "dct:related")
                      :as "agendapoint")
)
```
Another option could be having 3 different relationships (before, after, during) on the intermission and linking directly to the agendapoint. I think that might make the most sense from a data standpoint, though quite annoying to work with from a practical standpoint.

still needs to mature a bit in my head :).